### PR TITLE
fix: captured and added FileInfo query costs to metrics and hbar ratelimit class

### LINKED ISSUE #2738: Capture and add FileInfo query costs to metrics and hbar rate limiter class
### Problem

Currently. the cost of FileInfo queries in submitEthereumTransaction() flow is not captured in metrics and rate limit class.  

### Solution

Capture cost of FileInfo queries in metrics and rate limit class

### Alternatives

_No response_

### PR DESCRIPTION
**Pull request by quiet-node**
**Description**:
captured and added FileInfo query costs to metrics and hbar ratelimit class

**Related issue(s)**:

Fixes #2738

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

**Review by acuarica — APPROVED** [review]
lg

File: packages/relay/src/lib/clients/sdkClient.ts [hunk 1]
```diff
@@ -604,64 +604,60 @@ export class SDKClient {
     requestId?: string,
   ): Promise<T> {
     const requestIdPrefix = formatRequestIdMessage(requestId);
-    const currentDateNow = Date.now();
+    const queryType = query.constructor.name;
+    let queryResponse: any = null;
+    let queryCost: number | undefined = undefined;
```
Comment (ebadiere) [inline packages/relay/src/lib/clients/sdkClient.ts:609]: Nit: Does this need to be initialized to `undefined`?

**Review by ebadiere — COMMENTED** [review]
Good work.  One nit and a code coverage warning.  Can that be covered by a test?

File: packages/relay/src/lib/clients/sdkClient.ts (hunk 1 above)
Comment (quiet-node) [reply packages/relay/src/lib/clients/sdkClient.ts:609]: Ah yeah its a good point but I initilized to undefined as later the queryCost is assigned to `query._queryPayment?.toTinybars().toNumber()` which can be a number or undefined.

**Comment by quiet-node** [comment]
> Good work. One nit and a code coverage warning. Can that be covered by a test?

The additional code was added to pretty much modify the code layout, but the overall flow remains unchanged. There are already acceptance and unit tests for that. I believe the warning is sort of based on only to the newly added code.

**Review by ebadiere — APPROVED** [review]

**Pull request merged**